src/App.js

```diff
@@ -1,27 +1,42 @@
-import logo from './logo.svg';
 import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Col, Row } from 'react-bootstrap';
+import InputBar from './Components/InputBar';
+import SpeedBar from './Components/SpeedBar';
+import Clock from './Components/Clock';
+import PausePlayButton from './Components/PausePlayButton';
+import ClockComment from './Components/ClockComment';
 
-/** function responsible for generating the whole app
- * @returns {Node} displaying the UI
+/** This function renders the entire app
+ * @returns {Node} containing the app
  */
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Col xs={11} md={6} className="m-auto my-4 my-xl-5">
+      <Row>
+        <Col xs={12} md={10} className="m-auto ms-md-0">
+          <InputBar />
+        </Col>
+      </Row>
+      <Row className="mt-4">
+        <Col xs={12} md={10}>
+          <ClockComment />
```

Comment:
@tepeszCodes , doesn't this require props?

Reply:
Of course it does, I overlooked it, fixed it now